flowmachine/tests/test_active_subscribers.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from flowmachine.features.subscriber.active_subscribers import ActiveSubscribers
+from flowmachine.features.subscriber.unique_active_subscribers import (
+    UniqueActiveSubscribers,
+)
+from datetime import date, datetime
+from flowmachine.core.context import get_db
+
+
+@pytest.fixture()
+def active_sub_test_data(test_events_table):
```

Comment:
I don't like this.

I get the convenience of doing it, but, if every test did it because it was convenient then the already huge runtime of the test suite would get untenable, and I _don't_ see the necessity - having verified (and if necessary cos it is doing something complicated, demonstrated), that the query does what it should and fails as one would expect etc. then it doesn't matter that the test data doesn't show a realistic case, only that it continues to behave the same way and give the same result.

Reply:
I'll drop the fixture and replace it with the regular test_data before merge, but I'm planning on keeping it till then as a dev tool - it's already helped me catch two errors in `home_location`.
I get your concerns about test suite runtime - I tried to build the parent fixture with `scope=session` to avoid repeated CREATE, INSERT and DELETE,  but that involved creating a factory class for this, and then you need to design a bespoke dataset that is useful for every test it's used for and _then_ you've fallen down the artificial test data hole again.